tests/main/security-private-tmp/task.yaml

```diff
@@ -8,25 +8,26 @@ environment:
 
 prepare: |
     echo "Given a basic snap is installed"
-    . $TESTSLIB/snaps.sh
+    . "$TESTSLIB/snaps.sh"
     install_local test-snapd-tools
 
     echo "And another basic snap is installed"
-    mkdir -p $SNAP_INSTALL_DIR
-    cp -ra $TESTSLIB/snaps/test-snapd-tools/* $SNAP_INSTALL_DIR
-    sed -i 's/test-snapd-tools/not-test-snapd-tools/g' $SNAP_INSTALL_DIR/meta/snap.yaml
-    snapbuild $SNAP_INSTALL_DIR .
+    mkdir -p "$SNAP_INSTALL_DIR"
+    cp -ra "$TESTSLIB"/snaps/test-snapd-tools/* "$SNAP_INSTALL_DIR"
+    sed -i 's/test-snapd-tools/not-test-snapd-tools/g' "$SNAP_INSTALL_DIR/meta/snap.yaml"
+    snapbuild "$SNAP_INSTALL_DIR" .
     snap install --dangerous not-test-snapd-tools_1.0_all.snap
 
 restore: |
-    rm -rf not-test-snapd-tools_1.0_all.snap \
-          $SNAP_INSTALL_DIR /tmp/foo *stat.error
+    rm -rf not-test-snapd-tools_1.0_all.snap "$SNAP_INSTALL_DIR" /tmp/foo *stat.error
 
 execute: |
+    . "$TESTSLIB/dirs.sh"
+
     echo "When a temporary file is created by one snap"
-    expect -d -f tmp-create.exp
+    expect -d -f tmp-create.exp "$SNAPMOUNTDIR"
```

Comment:
As we discussed in irc, i'll work after this branch to unify the "_" convention for variables